balancer/xds/edsbalancer/util_test.go

```diff
@@ -20,8 +20,49 @@ package edsbalancer
 
 import (
 	"testing"
+
+	"google.golang.org/grpc/balancer/internal/wrr"
 )
 
+// testWRR is a deterministic WRR implementation.
+//
+// With {a: 2, b: 3}, the Next() results will be {a, a, b, b, b}.
+type testWRR struct {
+	itemsWithWeight []struct {
```

Comment:
You can reuse the `randWeighted` struct once you change the name.

Reply:
They are in different packages.